job-server/src/main/resources/application.conf

```diff
@@ -130,6 +130,9 @@ spark {
     # Start embedded H2 Server (usefull in cluster deployment)
     startH2Server = false
 
+    # Health Check Class to be invoked for healthz API
+    healthcheck = spark.jobserver.APIHealthCheck
```

Comment:
What is your opinion on putting this HealthCheck class in utils package?
`spark.jobserver.util.....`

Reply:
Addressed